<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DifferWidget</class>
 <widget class="QWidget" name="DifferWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>692</width>
    <height>325</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="1">
      <widget class="QLineEdit" name="first_snapshot_edit"/>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="second_snapshot_edit"/>
     </item>
     <item row="2" column="2">
      <widget class="QToolButton" name="second_snapshot_browser_button">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="first_snapshot_label">
       <property name="text">
        <string>1st snapshot:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QToolButton" name="first_snapshot_browser_button">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="second_snapshot_label">
       <property name="text">
        <string>2st snapshot:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="version_label">
       <property name="text">
        <string>version:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1" colspan="2">
      <widget class="QComboBox" name="version_edit"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTreeView" name="diff_view_tree"/>
   </item>
   <item>
    <widget class="ConsoleWidget" name="widget" native="true"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="topMargin">
      <number>0</number>
     </property>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="diff_button">
       <property name="text">
        <string>Diff</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ConsoleWidget</class>
   <extends>QWidget</extends>
   <header>console_widget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
